samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/ChatHistoryController.cs

```diff
@@ -28,23 +29,27 @@ public class ChatHistoryController : ControllerBase
     private readonly ChatSessionRepository _chatSessionRepository;
     private readonly ChatMessageRepository _chatMessageRepository;
     private readonly PromptsOptions _promptOptions;
+    private readonly ChatMemorySourceRepository _chatMemorySourceRepository;
```

Comment:
[nit] This could be simplified to `_memorySourceRepository` or even `_sourceRepository`.

Reply:
Same [nit] for the classname.

Reply:
should I change the others as well you think? wondering if we aim for consistency or correctness

Reply:
I changed it 😄